src/store/modules/mixes.js

```diff
@@ -107,7 +107,7 @@ const getters = {
 
 const actions = {
 	async fetchMixes({ commit }, id) {
-		let response = await axios.get("http://localhost:3000/mixes");
+		let response = await axios.get("https://mongo-illucid.herokuapp.com/mixes");
```

Comment:
Best practice is to always put these urls into a .env file
For example the env file would have: 
``` ILLUCID_BACKEND_URL=https://mongo-illucid.herokuapp.com```
And then a config file would be set up to load in all of your env variables.
`const ILLUCID_BACKEND_URL = process.env.ILLUCID_BACKEND_URL;`  (exporting those)

And then you would be able to use that variable in any axios call.
```let response = await axios.get(`${ILLUCID_BACKEND_URL}/mixes`)``` 
Note: all axios calls should exist within a function inside of an api folder's file - with try catches around the axios call to protect against the many unknowns that exist out there.